### PR TITLE
Fixed Phanbook\Models\ModelBase::getPostsWithVotes

### PR DESCRIPTION
**Pull request by sergeyklay**
Refs: #81

### CODE REVIEW

File: core/modules/frontend/controllers/PostsController.php [hunk 1]
```diff
@@ -335,7 +335,7 @@ public function newAction()
     public function viewAction($id, $slug)
     {
         $id     = (int) $id;
-        $userId = $this->auth->getAuth()['id'];
+        $userId = $this->auth->getUserId();
```
Comment (sergeyklay) [inline core/modules/frontend/controllers/PostsController.php:338]: @duythien `Auth::getAuth()` for anonymous returns null

File: core/common/models/ModelBase.php [hunk 2]
```diff
@@ -97,13 +98,13 @@ public function getVotes($objectId, $object)
...
-            (SELECT COALESCE(SUM(v.negative),0) FROM  vote v  WHERE p.id = v.objectId AND v.object = ?) AS negative
+            (SELECT COALESCE(SUM(v.positive),0) FROM vote v WHERE p.id = v.objectId AND v.object = ?) AS positive,
+            (SELECT COALESCE(SUM(v.negative),0) FROM vote v WHERE p.id = v.objectId AND v.object = ?) AS negative
             FROM postsReply p
             LEFT JOIN postsReplyHistory pr ON p.id = pr.postsReplyId
             LEFT JOIN users u ON u.id = p.usersId
-            WHERE p.postsId= ? AND p.deleted = 0
-            GROUP BY p.id
+            WHERE p.postsId = ? AND p.deleted = 0
+            GROUP BY p.id, pr.usersId
```
Comment (sergeyklay) [inline core/common/models/ModelBase.php:107]: @duythien This fixed MySQL 5.7 issue

Comment (duythien) [reply core/common/models/ModelBase.php:107]: That is great

File: core/modules/frontend/controllers/PostsController.php (hunk 1 above)
Comment (duythien) [reply core/modules/frontend/controllers/PostsController.php:338]: Ok I see my  problem now you fix it

File: core/config/services.php [hunk 3]
```diff
@@ -339,35 +339,42 @@ function () {
...
-    },
-    true
+        $path = ROOT_DIR . '/core/lang/messages/' . $code . '.php';
+        if (!file_exists($path)) {
+            $this->getLogger()->error("You must specify a language file for language '$code'");
+            $path = ROOT_DIR . '/core/lang/messages/en.php';
+        }
+
+        /** @noinspection PhpIncludeInspection */
+        $data = include $path;
```
Comment (sergeyklay) [inline core/config/services.php:368]: @duythien 

```
root@6b7e656b2129:/project# php -a
Interactive mode enabled

php > $data = require_once('/project/core/lang/messages/en.php');
php > echo gettype($data);
array
php > $data = require_once('/project/core/lang/messages/en.php');
php > echo gettype($data);
boolean
```

So when we run functional tests, the translation data is always `false`.

File: tests/functional/PostRepliesCept.php [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+$postId = $I->havePost(['usersId' => $userId]);
+
+$userId = $I->haveUser();
+$I->haveReply(['postsId' => $postId, 'usersId' => $userId]);
+
+$userId = $I->haveUser();
+$I->haveReply(['postsId' => $postId, 'usersId' => $userId]);
+
+$I->amOnPage("/questions/{$postId}/slug");
+$I->see('Answers ( 2 )');
```
Comment (sergeyklay) [inline tests/functional/PostRepliesCept.php:21]: @duythien Test for #81

File: core/common/models/PostsNotifications.php [hunk 5]
```diff
@@ -13,12 +13,12 @@
...
 
 /**
- * Postsnotifications
+ * \Phanbook\Models\PostsNotifications
  *
  * @autogenerated by Phalcon Developer Tools
  * @date          2015-07-29, 14:56:07
  */
-class Postsnotifications extends ModelBase
+class PostsNotifications extends ModelBase
```
Comment (sergeyklay) [inline core/common/models/PostsNotifications.php:21]: @duythien It works on MacOS, but not working in Linux (for example Travis CI uses Ubuntu)

**Pull request merged**